### PR TITLE
Add asreview2 base models support

### LINKED ISSUE #11: Add support for Logistic, SVM, and RF
- Also support the different optimization params for these models

### PR DESCRIPTION
**Pull request by timovdk**
Closes #11

### CODE REVIEW

File: asreview2-opt/main.py [hunk 1]
```diff
@@ -11,14 +11,24 @@
...
 from asreview.models.query import MaxQuery
 
 # Study variables
 PICKLE_FOLDER_PATH = Path("synergy-dataset", "pickles")
-STUDY_NAME = "ASReview2.x " + datetime.now().strftime("%Y-%m-%d at %H.%M.%S")
 N_STUDIES = 260
+CLASSIFIER_TYPE = (
+    "naive_bayes"  # Options: "naive_bayes", "logistic", "svm", "random_forest"
+)
+STUDY_NAME = (
```
Comment (J535D165) [inline asreview2-opt/main.py:29]: Maybe only the date here.

**Comment by timovdk** [comment]
I am open to suggestions about the parameters per model, and their ranges/categorical values. For example, there are more kernels than linear and rbf for SVM, but I am not sure which ones we want to optimize.

File: asreview2-opt/models.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        n_estimators=params["n_estimators"], max_features=params["max_features"]
+    )
+
+
+optuna_studies_models = {
+    "nb": naive_bayes_model,
+    "log": logistic_model,
+    "svm": svm_model,
+    "rf": random_forest_model,
+}
```
Comment (J535D165) [inline asreview2-opt/models.py:78]:
```suggestion

optuna_studies_models = {
    "nb": NaiveBayesClassifier,
    "log": LogisticClassifier,
    "svm": SVMClassifier,
    "rf": RandomForestClassifier,
}
```

File: asreview2-opt/main.py [hunk 3]
```diff
@@ -62,16 +63,18 @@ def run_sequential(studies, *args, **kwargs):
...
 
     priors = row["prior_inclusions"] + row["prior_exclusions"]
 
-    # Create classifier and balancer with optuna value
-    clf = NaiveBayesClassifier(alpha=alpha)
+    # Create balancer with optuna value
     blc = Balanced(ratio=ratio)
 
+    # Create classifier with params
+    clf = optuna_studies_models[CLASSIFIER_TYPE](params)
```
Comment (J535D165) [inline asreview2-opt/main.py:76]:
```suggestion
    clf = optuna_studies_models[CLASSIFIER_TYPE](**params)
```

**Comment by J535D165** [comment]
Yeah, we have to read a bit more literature on this. For now, I think we have a nice hyperparam space.

**Comment by J535D165** [comment]
Feel free to merge. Don't forget about the ratio param space before you start the machines!

**Pull request merged**